test/rules/typedef-whitespace/both/onespace/test.ts.lint

```diff
@@ -220,7 +220,7 @@ function withPreceedingSpaceNoSubsequentSpaceFn(a :number, b :number) :number {
     }
 }
 
-function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b : number) : number {
+function withPreceedingSpaceWithSubsequentSpaceFn(a : number, b ?: number) : number {
```

Comment:
Ah, I see what you're getting at. So with your tweaks, the rule would allow either `foo? : number` or `foo ?: number`? (Assuming the settings are for one space on both sides of the colon.)